grpc-service/Dockerfile

```diff
@@ -2,5 +2,5 @@ FROM openjdk:8
 
 ADD target/grpc-service-1.0-SNAPSHOT.jar /home/app.jar
 ADD rockyou.tar.gz /home
-
+CMD tar xf /home/rockyou.tar.gz /home
```

Comment:
I'm still afraid this may fail in linux .. would you please try the following:

1. Use COPY instead of ADD for the dict file
2. RUN tar -xvf COMPRESSED_FILE /home &>/dev/null

This should work in all unix environments